pallets/stake/src/lib.rs

```diff
@@ -14,7 +14,30 @@
 // You should have received a copy of the GNU General Public License
 // along with Moonbeam.  If not, see <http://www.gnu.org/licenses/>.
 
-//! Minimal staking module with ordered validator selection
+//! # Stake
+//! Minimal staking pallet that implements ordered validator selection by total amount at stake
+//!
+//! ### Rules
+//! There is a new round every `BlocksPerRound` blocks.
+//!
+//! At the start of every round,
+//! * `IssuancePerRound` is distributed to validators for `BondDuration` rounds ago
+//! in proportion to the points they received in that round (for authoring blocks)
+//! * queued validator exits are executed
+//! * new set of validators is chosen from the candidates
+//!
+//! To join the set of candidates, an account must call `join_candidates` with
+//! stake >= `MinValidatorStk` and fee <= `MaxFee`. The fee is taken off the top
+//! of any rewards for the validator before the remaining capital is distributed to nominators in
+//! proportion to stake (and the validator always self-nominates)
```

Comment:
So the validator get `fee` + `percent of rewards based on its bond` right ?

Reply:
Yes, correct